app/resources/create.html.template

```diff
@@ -572,11 +571,9 @@
 
         </div>
       </div>
-      </div>
 
-      <div class="column end"></div>
       <div class="end-multi-columns"></div>
```

Comment:
You probably don't need this div element because it's no longer multi-column. Can you try to delete it and let me know if it breaks the layout?